web/src/features/fba/components/ZoneSummaryPanel.tsx

```diff
@@ -34,40 +35,37 @@ interface Props {
   fireZoneAreas: FireZoneArea[]
 }
 
-const ZoneSummaryPanel = (props: Props) => {
+const ZoneSummaryPanel = React.forwardRef((props: Props, ref: React.ForwardedRef<HTMLDivElement>) => {
   const classes = useStyles()
+  ZoneSummaryPanel.displayName = 'ZoneSummaryPanel'
```

Comment:
I think we can delete this
```suggestion
```

Reply:
I wasn't confident about what I did here, but it was a solution to prevent the linter complaining hard about the Component definition missing a display name, which from what I've read could be helpful for debugging?

Another solution would be add a `/* eslint-disable react/display-name */` 

Or maybe there's a better solution...

Reply:
Huh weird, nevermind I think this is correct. It's expected by React forward ref components I think